app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def create
+    user = current_user
+    post = params[:post_id]
+    @comment = Comment.new(
+      text: params[:text],
+      post_id: post,
+      author_id: user.id
+    )
+    return unless @comment.save
+
+    redirect_to user_post_path(id: post, user_id: user.id)
+  end
+end
```

Comment:
- [ ] It would be better if we create the `new` method for our comment controller action. The new method returns to us a new object

Reply:
> @addod19 I didn't get you isn't that what the create method does can you elaborate

No please, since we have the create method we should also have the new method. Meaning we should both create and new method according to the project requirements

Reply:
> @addod19 I didn't get you isn't that what the create method does can you elaborate

@Cluab, create method saves object into the database whilst the new method returns a new object

Reply:
@addod19 there is nothing of that such in the project requirements:face_exhaling: 
![Screenshot from 2023-03-02 19-40-12](https://user-images.githubusercontent.com/103601335/222508812-a5e446f2-5c9d-48ad-ba4c-d85528452e37.png)

Reply:
@addod19 can you please explain what does the new method do and what is the difference from it and the create method